static/app/components/replays/utils.tsx

```diff
@@ -122,7 +122,12 @@ export function getCrumbsByColumn(
     const sinceStart = isNaN(timestampMilliSeconds)
       ? 0
       : timestampMilliSeconds - startMilliSeconds;
-    const column = Math.floor((sinceStart / safeDuration) * (totalColumns - 1)) + 1;
+
+    const columnPositionCalc =
+      Math.floor((sinceStart / safeDuration) * (totalColumns - 1)) + 1;
+
+    // Should start at minimum in the first column
+    const column = columnPositionCalc <= 0 ? 1 : columnPositionCalc;
```

Comment:
is this the same as using max?

```suggestion
    const column = Math.max(1, columnPositionCalc);
```